.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+## Checklist
+___Do not mention proprietary info or link to internal work items in this PR.___
+
+**PR Buddy:** _@ mention a team member_
+
+**What were the changes?**  
+_One sentence describing the work done._
+
+**Why were the changes made?**  
+_Explain the motivation behind the work. Provide historical context and customer-specific info._
+
+**How was the outcome achieved?**  
+_Technical details behind the work. Explain any AMD hardware gotchas._
+
+**Additional Documentation:**  
+_What else should the reviewer know?_
+
+**Test Cases:**
+- [ ] _List here_
+
+## Approval Checklist
+- [ ] PR Buddy: Test functionality
+- [ ] PR Buddy: Test performance
+- [ ] PR Buddy: Verify accurate documentation by testing listed steps
+- [ ] Ensure checklist criteria are met
+- [ ] Ensure CI passes
```

Comment:
This needs to be moved to before merging

Reply:
The entire approval checklist is meant to be performed by the maintainers before merging. Perhaps there should be a pre-merge and post-merge checklist?

Reply:
maybe approval checklist and merging checklist?
We normally always leave merging to author as he/she has most updated knowledge on a PR.

Reply:
So far, the only item on the merge checklist is to wait for approval, so I'm going to leave that off for now. All approval steps will be intended for before merging.